specs/deliverly_server_SUITE.erl

```diff
@@ -85,6 +111,50 @@ auth_failed(_) ->
   1 = Size,
   ok.
 
+auth_token_success(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{})),
+  Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
+  {reply, _, _} = Res,
+  ok.
+
+auth_token_failed(_) ->
+  Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, <<"123">>}]),
+  {error, 3401} = Res,
+  ok.
+
+auth_token_timeout(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{expires_in => 1})),
+  timer:sleep(1000),
+  Res = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
+  {error, 3401} = Res,
+  ok.
+
+auth_token_once(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{expires_in => 10, once => true})),
+  Res1 = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
+  {reply, _, _} = Res1,
+  Res2 = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
+  {error, 3401} = Res2,
+  ok.
+
+auth_token_infinity(_) ->
+  Token = proplists:get_value(token, de_auth_token:request_token(#{expires_in => infinity})),
+  Res1 = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
+  {reply, _, _} = Res1,
+  Res2 = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),
```

Comment:
Line 144 is too long:   Res2 = deliverly_server:auth_client(#de_client{socket=1, app = default}, [{<<"token">>, Token}]),.